pkg/vm/stackitem/item.go

```diff
@@ -799,6 +799,25 @@ type MapElement struct {
 	Value Item
 }
 
+// ToStackItem implements convertible interface and is used for emitter compatibility.
```

Comment:
I don't think you really need it.

Reply:
With `PACKMAP` we don't need it, that's true.